judging/scripts/bootstrap.sh

```diff
@@ -33,11 +40,20 @@ archive_file="$HOME/$repo_name.tar.gz"
 wget --quiet --output-document "$archive_file" https://github.com/leaf-ai/$repo_name/archive/$branch.tar.gz
 
 # Unzip to destination directory
-mkdir -p "$repo_dir" && \
```

Comment:
PyCharm knew something I didn't -- you don't need line continuation after certain incantations in bash, `&&` being one of them. I didn't notice it had removed the backslashes here but it's legit.